docker/nginx/conf.d/client.conf

```diff
@@ -108,6 +108,8 @@ server {
 	location /health-check {
 		include /etc/nginx/conf.d/include/cors;
 
+		access_log off; # do not log traffic to health-check endpoint
```

Comment:
health check dashboard is based on this, I'd rather include it and exclude it manually from graphs, it's pretty easy to exclude